python/src/worker/tasks/background_expressed_genes.py

```diff
@@ -50,15 +51,23 @@ def compute(self):
         request = self._format_request()
 
         # send request to r worker
-        r = requests.post(
+        response = requests.post(
             f"{config.R_WORKER_URL}/v0/getBackgroundExpressedGenes",
             headers={"content-type": "application/json"},
             data=json.dumps(request),
         )
 
-        # raise an exception if an HTTPError if one occurred because otherwise r.json()
+        # raise an exception if an HTTPError if one occurred because otherwise response.json()
```

Comment:
could you fix the comment? remove the extra "if one"  `raise an exception if an HTTPError **if one** occurred`

Reply:
done 👍